lib/mongoose_push_web/plugs/maybe_put_swagger_ui.ex

```diff
@@ -0,0 +1,30 @@
+defmodule MongoosePushWeb.Plug.MaybePutSwaggerUI do
+  import Plug.Conn
+  @behaviour Plug
+
+  @impl Plug
+  def init(opts) do
+    enable =
+      Application.get_env(:mongoose_push, :openapi)[:expose_spec] &&
```

Comment:
Maybe it would be cleaner to fetch `Application.get_env(:mongoose_push, :openapi)` once, assign it and just access sub-fields twice?